source/django_package/domain/import_datas/import_csv/import_csv_data.py

```diff
@@ -0,0 +1,8 @@
+import requests
+
+from source.django_package.domain.import_datas import parsing_config_file
+
+
+def get_data():
+    r = requests.get(parsing_config_file.ReadConfigFile.data_source[0])
+    print(r.text)
```

Comment:
Not finished yet, right? I think it should use the adapter to obtain the list of columns that we need (we might want to skip some of them), parse the response content (deserialize it) and return as an object (could be list of model instances later on, when we have models).
`r` is not a good variable name. Use something more descriptive.
How about errors? They do happen, especially network/connection errors - network could be down, the file might be moved somewhere else etc. Consider what you want to do about it. In case of connection errors, you might want to wait few seconds and try again.